app/controllers/wpcc/engine_controller.rb

```diff
@@ -1,9 +1,15 @@
 module Wpcc
   class EngineController < Wpcc.parent_controller.constantize
+    WPCC_SESSION_EXPIRY_LIMIT = (30 * 60)
```

Comment:
I've got it now. But On the yml file you can do: 

```
  session_expiration_time: <%= 30.minutes %>
```

OR since you're multiplying the value by 60 you always need to use minutes. In case is the two hours the expiration will be 120 minutes, so I can't see no problem using .minutes.